### PR TITLE
FUSETOOLS-2435: Migrate SAP QE tests to jbosstools-fuse-extras repo

### PR DESCRIPTION
**Pull request by apodhrad**

### CODE REVIEW

**Comment by apodhrad** [comment]
This PR can be merged after https://github.com/jbosstools/jbosstools-fuse/pull/998

**Comment by apodhrad** [comment]
I need to update the tests for RedDeer 2.0

File: jboss-fuse-sap-tool-suite/uitests/tests/org.jboss.tools.fuse.sap.ui.bot.tests/launcher/RedDeer Test Linux x86_64.launch [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (apupier) [inline jboss-fuse-sap-tool-suite/uitests/tests/org.jboss.tools.fuse.sap.ui.bot.tests/launcher/RedDeer Test Linux x86_64.launch]: Is it possible to use a non-platform specific launch configuration?

Comment (apodhrad) [reply jboss-fuse-sap-tool-suite/uitests/tests/org.jboss.tools.fuse.sap.ui.bot.tests/launcher/RedDeer Test Linux x86_64.launch]: yes, I'm already working on it

**Comment by apodhrad** [comment]
The PR is ready

**Comment by apodhrad** [comment]
Note that we cannot use
```
-DskipTests=false
```
otherwise the UI tests will be executed. At the moment don't want to execute such tests for each build.

**Review by apupier — APPROVED** [review]

**Pull request merged**